controllers/gcpcluster_controller.go

```diff
@@ -50,7 +50,7 @@ func (r *GCPClusterReconciler) SetupWithManager(mgr ctrl.Manager, options contro
 		WithOptions(options).
 		For(&infrav1.GCPCluster{}).
 		Watches(
-			&source.Kind{Type: &infrav1.GCPCluster{}},
+			&source.Kind{Type: &infrav1.GCPMachine{}},
```

Comment:
I guess this handler was just failing until now :p

Reply:
Correct, I only noticed when I switched to testing and troubleshooting locally.